### PR TITLE
create authorization with authentication ,add custom ExceptionHandler

### PR DESCRIPTION
**Pull request by metrondir**
Now in this project realize:

Authorization and Authentication using Identity
Email sending to confirm user registration
Logic of refreshToken and accessToken
Custom exception handler to manage exceptions well

### CODE REVIEW

File: SoftServerCinema.Security/Controllers/UserController.cs [hunk 1]
```diff
@@ -28,7 +72,73 @@ public async Task<IActionResult> Register([FromBody] UserRegisterDTO userRegiste
             if (user)
                 return Ok();
             else
-                return BadRequest();
+                throw new ApiException()
+                {
+                    StatusCode = StatusCodes.Status500InternalServerError,
+                    Title = "User creation",
```
Comment (Zorro769) [inline SoftServerCinema.Security/Controllers/UserController.cs]: Change to "User creation failed"

**Review by Zorro769 — CHANGES_REQUESTED** [review]
Fix all comments

File: SoftServerCinema.Security/Controllers/UserController.cs (hunk 1 above)
Comment (metrondir) [reply SoftServerCinema.Security/Controllers/UserController.cs]: resolved!

**Review by Zorro769 — APPROVED** [review]
Everything is fine

**Pull request merged**